packages/core/src/components/checkbox/checkbox.tsx

```diff
@@ -185,6 +185,7 @@ export class Checkbox implements IxFormComponent<string> {
       <Host
         aria-checked={a11yBoolean(this.checked)}
         aria-disabled={a11yBoolean(this.disabled)}
+        aria-label={this.label}
```

Comment:
aria-label can be removed its not necessary for the accessibility tree
<img width="599" height="323" alt="image" src="https://github.com/user-attachments/assets/5e15cc74-8c25-40d0-8052-b405a8a8aedf" />

Reply:
removed. thanks